tox.ini

```diff
@@ -2,7 +2,7 @@
 distshare = {homedir}/.tox/distshare
 envlist = py38-pytest{46,50,51,52,53,54,60,61,62,latest,master},
           py38-pytestlatest-linters,
-          py{36,37,39}-pytestlatest
+          py{36,37,39,310}-pytestlatest
```

Comment:
python 3.10 is not stable yet, so please let's add `ignore-outcome = true` for env with `py310` (like it's done for `py38-pytestmaster` [here](https://github.com/pytest-dev/pytest-factoryboy/blob/1601cb794c703d6f6d442ae1ab674cbb973e1055/tox.ini#L31))